README.md

```diff
@@ -588,14 +606,16 @@ To begin using the `TargetingFilter` in an application it must be added to the a
 
 The implementation type used for the `ITargetingContextAccessor` service must be implemented by the application that is using the targeting filter. Here is an example setting up feature management in a web application to use the `TargetingFilter` with an implementation of `ITargetingContextAccessor` called `HttpContextTargetingContextAccessor`.
 
-```
-services.AddFeatureManagement();
+``` C#
+services.AddFeatureManagement()
         .WithTargeting<HttpContextTargetingContextAccessor>();
 ```
 

```

Comment:
> The targeting context accessor and targeting filter will be registered as the same type as the feature manager.

I'd like this topic to stick in the single section discussing service lifetime. As long as we say there a statement that all filters are added with the same service lifetime then we ashould be covered.